test/unit/pages/patientnew.test.js

```diff
@@ -1,21 +1,15 @@
-/* global beforeEach */
-/* global before */
-/* global after */
 /* global chai */
-/* global describe */
 /* global sinon */
-/* global it */
 
 import _ from 'lodash';
 import React from 'react';
 import TestUtils from 'react-dom/test-utils';
 import mutationTracker from 'object-invariant-test-helper';
 import { mount } from 'enzyme';
 
-import { PatientNew } from '../../../app/pages/patientnew';
-import { mapStateToProps } from '../../../app/pages/patientnew';
+import { PatientNew, mapStateToProps } from '../../../app/pages/patientnew/patientnew';
 
-import i18n from 'i18next';
+import * as i18n from 'i18next';
```

Comment:
I don't know why you have to change this import, it works at is.
```js
import i18n from 'i18next';
```
But if you do you can place it before the "local" import.
It's always better to do them in order, eg: external import first, local after.
It is to avoid some init side effects.

Reply:
eslint whines when I  keep `import i18n from 'i18next';`

```
119:9  warning  Caution: `i18n` also has a named export `off`. Check if you meant to write `import {off} from 'i18next'` instead                        import/no-named-as-default-member
120:9  warning  Caution: `i18n` also has a named export `changeLanguage`. Check if you meant to write `import {changeLanguage} from 'i18next'` instead  import/no-named-as-default-member
126:9  warning  Caution: `i18n` also has a named export `changeLanguage`. Check if you meant to write `import {changeLanguage} from 'i18next'` instead  import/no-named-as-default-member 
```
Just moving it to before local declarations